reactioncommerce/reaction
# Setup migration path from default roles to groups

Migration path from previous permission management to permission groups.

To Test:
* Run reaction on development branch, create a few users, and promote a few to admin (i.e giving them "dashboard" and some other roles.
* Switch to this branch (seun-migration-to-groups)
* After a few seconds (or minutes), the app restarts and runs the migration. (NB: There might be errors of missing packages. Install them. There's a possibility of getting a `toString`ed, and you'll have to restart.
* Once the app has restarted, check to see that default groups have been created. Check that user accounts that are part of default groups have been assigned to those groups.
* Check that custom groups were created, and users belonging to them have been assigned (NB: Only Admin Groups will show up in the Admin Accounts dashboard).
